### ISSUE TITLE
✅ 🐞 Bug Report: Gallery widget lightbox omits navigation links - [ED-11146]

### ISSUE THREAD

**Issue by caspervoogt** [elementor/elementor#22870]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I am using Elementor 3.14. When I use a Gallery widget (available in Pro only) and enable the lightbox option, the lightbox includes no navigation (neither arrows nor dots). I worked around the issue by switching to a Basic Gallery widget (available in Elementor without pro), because its lightbox DOES include navigation links. 

The Gallery widget's lightbox should also display navigation links.

I did also tested this on a fresh WordPress installation with just Elementor and Elementor Pro (3.14) and was able to reproduce this.

### Steps to reproduce

Add Gallery widget with lightbox enabled. I tested with Elementor and with Elementor Pro and it happened in both cases.

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.57 (Debian)
	MySQL version: mariadb.org binary distribution v10.4.20
	PHP Version: 8.1.20
	PHP Memory Limit: 512M
	PHP Max Input Vars: 2500
	PHP Max Post Size: 10M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.2
	Site URL: https://wordpress-614902-1994353.cloudwaysapps.com
	Home URL: https://wordpress-614902-1994353.cloudwaysapps.com
	WP Multisite: No
	Max Upload Size: 10 MB
	Memory limit: 512M
	Max Memory limit: 512M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en-US
	Timezone: 0
	Admin Email: [REDACTED]
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36

== Active Plugins ==
	Admin Taxonomy Filter
		Version: 1.0.2
		Author: eLightUp

	Advanced Custom Fields
		Version: 6.1.6
		Author: WP Engine

	Advanced Custom Fields PRO
		Version: 6.1.6
		Author: WP Engine

	Breeze
		Version: 2.0.26
		Author: Cloudways

	Classic Editor
		Version: 1.6.3
		Author: WordPress Contributors

	Crop Thumbnails
		Version: 1.7.2
		Author: Volkmar Kantor

	Debug Bar
		Version: 1.1.4
		Author: wordpressdotorg

	Disable Comments
		Version: 2.4.4
		Author: WPDeveloper

	Duck Kint Debugger
		Version: 2.0.1
		Author: Brian Fegter, Chris Dillon

	DynamicConditions
		Version: 1.6.2
		Author: RTO GmbH

	DynamicTags
		Version: 1.2.2
		Author: RTO GmbH

	Ele Custom Skin
		Version: 3.1.7
		Author: Dudaster.com

	Elementor
		Version: 3.14.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.14.0
		Author: Elementor.com

	Essential Addons for Elementor
		Version: 5.8.0
		Author: WPDeveloper

	Essential Addons for Elementor - Pro
		Version: 5.4.11
		Author: WPDeveloper

	Highlight Search Terms
		Version: 1.8.2
		Author: RavanH

	Intuitive Custom Post Order
		Version: 3.1.4.1
		Author: hijiri

	MalCare WordPress Security Plugin - Malware Scanner, Cleaner, Security Firewall
		Version: 5.09
		Author: MalCare Security

	OoohBoi Steroids for Elementor
		Version: 2.1.5
		Author: OoohBoi

	Plethora Map
		Version: 
		Author: Jesse Voogt

	Redirection
		Version: 5.3.10
		Author: John Godley

	Regenerate Thumbnails
		Version: 3.1.5
		Author: Alex Mills (Viper007Bond)

	Search & Filter Pro
		Version: 2.5.14
		Author: Code Amp

	SVG Support
		Version: 2.5.5
		Author: Benbodhi

	WDP&A Custom Stuff
		Version: 
		Author: Casper Voogt

	WebP Express
		Version: 0.25.6
		Author: Bjørn Rosell

	Wordfence Login Security
		Version: 1.1.3
		Author: Wordfence


== Must-Use Plugins ==
	Object Cache Pro (MU)
		Version: 1.18.2
		Author: Rhubarb Group


== Elements Usage ==
	
	archive : 4
		archive-posts : 1
		heading : 7
		posts : 6
		text-editor : 1
	footer : 1
		eael-logo-carousel : 1
		heading : 3
		social-icons : 1
	header : 1
		image : 2
		nav-menu : 3
		search-form : 2
		social-icons : 1
	loop : 3
		button : 1
		eael-flip-box : 1
		image : 1
		post-info : 1
		theme-post-content : 1
		theme-post-featured-image : 2
		theme-post-title : 1
	wp-post : 
		text-editor : 3
		toggle : 1
	wp-page : 17
		blockquote : 3
		button : 6
		eael-google-map : 6
		eael-post-list : 2
		eael-testimonial-slider : 1
		flip-box : 1
		form : 2
		gallery : 1
		heading : 19
		icon-box : 7
		icon-list : 12
		image : 12
		posts : 2
		shortcode : 1
		spacer : 1
		text-editor : 9
	single-post : 4
		button : 1
		heading : 9
		image : 2
		image-carousel : 1
		image-gallery : 1
		shortcode : 1
		text-editor : 6
		theme-post-content : 3
		theme-post-featured-image : 6
		theme-post-title : 4
		toggle : 1
	single-page : 1
		theme-post-content : 1
		theme-post-title : 1
	search-results : 1
		archive-posts : 1
		search-form : 1
		theme-archive-title : 1


== Settings ==
	
	cpt_support: post, page, services
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	unfiltered_files_upload: 1


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	recaptcha_v3: Active


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Inactive by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Inactive by default
	Upgrade Swiper Library: Inactive by default
	Grid Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Inactive by default
	Editor Top Bar: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Inactive by default
	Nested Accordion: Inactive by default
	Lazy Load Background Images: Inactive by default
	Global Style Guide: Inactive by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default


== Log ==
	
Log: showing 20 of 352023-05-25 14:50:39 [info] elementor::elementor_updater Started 
2023-05-25 14:50:39 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-25 14:50:40 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-25 14:50:40 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.1',
  'to' => '3.13.3',
)]
2023-06-01 17:52:16 [info] elementor::elementor_updater Started 
2023-06-01 17:52:16 [info] Elementor/Upgrades - _on_each_version Start  
2023-06-01 17:52:16 [info] Elementor/Upgrades - _on_each_version Finished 
2023-06-01 17:52:16 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.3',
  'to' => '3.13.4',
)]
2023-06-01 17:52:16 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.3',
  'to' => '3.13.4',
)]
2023-06-21 16:02:28 [info] elementor-pro::elementor_pro_updater Started 
2023-06-21 16:02:28 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-06-21 16:02:28 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-06-21 16:02:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.2',
  'to' => '3.14.0',
)]
2023-06-21 16:02:28 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.2',
  'to' => '3.14.0',
)]
2023-06-21 16:02:28 [info] elementor::elementor_updater Started 
2023-06-21 16:02:28 [info] Elementor/Upgrades - _on_each_version Start  
2023-06-21 16:02:29 [info] Elementor/Upgrades - _on_each_version Finished 
2023-06-21 16:02:29 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.4',
  'to' => '3.14.0',
)]
2023-06-21 16:02:29 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.4',
  'to' => '3.14.0',
)]
2023-06-21 16:02:29 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.2',
  'to' => '3.14.0',
)]

PHP: showing 4 of 4PHP: 2023-04-13 13:49:25 [notice X 1610][/home/614902.cloudwaysapps.com/urqfneppmc/public_html/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::204] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-22 18:56:24 [warning X 13][/home/614902.cloudwaysapps.com/urqfneppmc/public_html/wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-gallery.php::36] Undefined array key 1 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-22 18:57:35 [error X 1][/home/614902.cloudwaysapps.com/urqfneppmc/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::304] Uncaught Error: Call to a member function update_meta() on null in /home/614902.cloudwaysapps.com/urqfneppmc/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:304
Stack trace:
#0 /home/614902.cloudwaysapps.com/urqfneppmc/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(177): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions()
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions()
#2 /home/614902.cloudwaysapps.com/urqfneppmc/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func()
#3 /home/614902.cloudwaysapps.com/urqfneppmc/public_html/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request()
#4 /home/614902.cloudwaysapps.com/urqfneppmc/public_html/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters()
#5 /home/614902.cloudwaysapps.com/urqfneppmc/public_html/wp-includes/plugin.php(517): WP_Hook->do_action()
#6 /home/614902.cloudwaysapps.com/urqfneppmc/public_html/wp-admin/admin-ajax.php(188): do_action()
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-22 19:20:09 [warning X 2][/home/614902.cloudwaysapps.com/urqfneppmc/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 3 of 3JS: 2023-06-20 13:25:42 [error X 1][https://wordpress-614902-1994353.cloudwaysapps.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.13.4:3:794943] Cannot read properties of undefined (reading 'toLowerCase') 
JS: 2023-06-20 13:29:46 [error X 2][https://wordpress-614902-1994353.cloudwaysapps.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.13.4:3:795450] Cannot read properties of undefined (reading 'localized') 
JS: 2023-06-21 19:39:16 [error X 1][https://wordpress-614902-1994353.cloudwaysapps.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.14.0:3:639878] Cannot read properties of undefined (reading 'isDesignable') 



== Elementor - Compatibility Tag ==
	
	Ele Custom Skin: Incompatible
	Elementor Pro: Compatible
	Essential Addons for Elementor: Incompatible
	Essential Addons for Elementor - Pro: Incompatible
	OoohBoi Steroids for Elementor: Incompatible

== Elementor Pro - Compatibility Tag ==
	
	Ele Custom Skin: Incompatible
	Essential Addons for Elementor: Incompatible
	Essential Addons for Elementor - Pro: Incompatible
	OoohBoi Steroids for Elementor: Incompatible
				

````

</details>

**Comment by CamelDesign10** [comment]
same here since the last update 3.14 gallery lightbox have : no arrow, no dot, no navigation by keybord .... no navigation at all
![Capture d’écran 2023-06-23 173323](https://github.com/elementor/elementor/assets/115035112/4bedc245-414c-43f5-ac2b-618e63b9c1d2)

**Comment by vgstef** [comment]
Yeah, very annoying! I need a masonry gallery, but no ways to cycle between images in the lightbox viewer. Can Elementor team fix this?

**Comment by nicholaszein** [comment]
Hello @caspervoogt! 

Thank you for submitting your issue! 🙏 

We are aware of this issue and our team is already working on a fix. We can’t say exactly when it'll be sorted out, but we kindly ask for your patience. Thanks for understanding! 😉

Best regards

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.14.1! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
